frontend/.env.example

```
@@ -4,8 +4,12 @@ EXPO_PUBLIC_TOKEN_URL=https://api.asgardeo.io/t/<asgardeo-organization>/oauth2/t
 EXPO_PUBLIC_LOGOUT_URL=https://api.asgardeo.io/t/<asgardeo-organization>/oidc/logout
 EXPO_PUBLIC_BACKEND_BASE_URL=<backend-url>         # Backend API Base URL
 
-EXPO_PUBLIC_LIBRARY_ARTICLE_FALLBACK_IMAGE=
-EXPO_PUBLIC_DEVELOPER_APP_DEFAULT_URL=
+#Please note that this URL is required only for the WSO2 Super App.
+EXPO_PUBLIC_LIBRARY_ARTICLE_FALLBACK_IMAGE=<fallback-image-url>
+
+#Please note that the developer app-related URL is required only for the WSO2 Super App. 
+EXPO_PUBLIC_DEVELOPER_APP_IOS_DEFAULT_URL=<developer-app-ios-url>
+EXPO_PUBLIC_DEVELOPER_APP_ANDROID_DEFAULT_URL=<developer-app-android-url>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_DEVELOPER_APP_ANDROID_DEFAULT_URL key should go before the EXPO_PUBLIC_DEVELOPER_APP_IOS_DEFAULT_URL key

(UnorderedKey)